web/renderer/pages/database/[databaseName]/pulls/compare/[refName]/[fromBranchName]/index.tsx

```diff
@@ -1,23 +1,21 @@
 import Page from "@components/util/Page";
-import { Maybe } from "@dolthub/web-utils";
-import { PullDiffParams } from "@lib/params";
+import { PullDiffParamsOptionalTableName } from "@lib/params";
 import DatabasePage from "@pageComponents/DatabasePage";
 import { GetServerSideProps, NextPage } from "next";
 
 type Props = {
-  params: PullDiffParams;
-  tableName?: Maybe<string>;
+  params: PullDiffParamsOptionalTableName;
```

Comment:
I think this should have a `null` tableName type so that you don't need to type cast below?